.env.example

```diff
@@ -13,3 +13,5 @@ DATA_INGESTION_API_KEY=
 
 CPU_TYPE=ONE_VCPU # ONE_VCPU, TWO_VCPU, FOUR_VCPU, QUARTER_VCPU, HALF_VCPU
 MEMORY_TYPE=TWO_GB # TWO_GB, FOUR_GB, EIGHT_GB, TWELVE_GB, ONE_GB, HALF_GB, SIX_GB
+
+VPC_ID=YOUR_VPC_ID
```

Comment:
**Suggestion:** Provide an example format for the VPC_ID in the .env.example file to guide users on how to correctly set this environment variable.
> 🌆  **Medium Priority** 
 > [category: **`documentation`**, importance: **`6`**]
> **Reason:** This suggestion enhances the example environment file, providing clearer guidance to users on how to correctly set the VPC_ID variable.
 

```suggestion
VPC_ID=vpc-0123abcd # Replace with your actual VPC ID (e.g., vpc-0123abcd)
```